Build failure: quickgui

### Steps To Reproduce

Steps to reproduce the behavior:
1. build *quickgui*

### Build log

https://gist.github.com/LuNeder/271b9a730e039e8e12a3296ddea6936a

### Additional context

```
Failed obtaining PRETTY_NAME for Linux: PathNotFoundException: Cannot open file, path = '/usr/lib/os-release' (OS Error: No such file or directory
, errno = 2)
[...]
 /nix/store/4wj8q7c8fjk9j7xw6iiz7r4qrpg6gc6y-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1915:39: Error: Method not found: 'SchemeRainbo
w'.
[        ] [   +1 ms]       DynamicSchemeVariant.rainbow => SchemeRainbow(sourceColorHct: sourceColor, isDark: isDark, contrastLevel: contrastLevel),
[        ] [        ]                                       ^^^^^^^^^^^^^
[        ] [        ] /nix/store/4wj8q7c8fjk9j7xw6iiz7r4qrpg6gc6y-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1916:42: Error: Method not found: 'SchemeFruitS
alad'.
```
### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
@flexiondotorg  @heyimnova 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 ✘ luana@Luana-X670E  ~  nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.10.6, NixOS, 24.11 (Vicuna), 24.11.20240821.c374d94`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - nixpkgs: `/nix/store/j8pbrsb3nybdap3hhg9kw0ffqd4rlbx6-source`


```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

> Nothing is blocking, just low on bandwidth for myself to create a PR. That error isn't a blocker and is non fatal, every Flutter build runs into that error.

This is showing up as a fatal error for me when trying to build after changing the revision. Full logs: 

```
[        ] [   +7 ms] Failed obtaining PRETTY_NAME for Linux: PathNotFoundException: Cannot open file, path = '/usr/lib/os-release' (OS Error: No such file or directory, errno = 2)
[        ] [   +9 ms] Unable to locate an Android SDK.
[        ] [  +63 ms] Initializing file store
[        ] [  +13 ms] Skipping target: gen_localizations
[        ] [   +2 ms] invalidated build due to missing files: /build/source/.dart_tool/package_config_subset
[        ] [   +1 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputMissing: The following inputs were missing: /build/source/.dart_tool/package_config_subset}
[        ] [   +4 ms] unpack_linux: Starting due to {}
[        ] [  +45 ms] unpack_linux: Complete
[        ] [ +145 ms] Found plugin shared_preferences_foundation at /nix/store/0sv2j1z4kmvghapw6zhpf611a3fj638v-pub-shared_preferences_foundation-2.4.0/
[        ] [   +6 ms] Found plugin window_size at /nix/store/281pj1qkmwsbzmn02r6q9icaj11x15ab-pub-window_size-0.1.0/plugins/window_size/
[        ] [   +4 ms] Found plugin shared_preferences_android at /nix/store/36mbsh7g74j65sc2mpflp1sgqykz6dbz-pub-shared_preferences_android-2.2.3/
[        ] [   +2 ms] Found plugin shared_preferences_linux at /nix/store/602xl16ciy9nh5v05yigvaabllifcgh7-pub-shared_preferences_linux-2.3.2/
[        ] [   +1 ms] Found plugin path_provider_linux at /nix/store/6r85azf4l96yfy0b950sbp90ygxiqap0-pub-path_provider_linux-2.2.1/
[        ] [   +2 ms] Found plugin url_launcher_linux at /nix/store/76zc8zsq3cvimmjivzmbivcc7zq6sm3i-pub-url_launcher_linux-3.1.1/
[        ] [   +9 ms] Found plugin package_info_plus at /nix/store/cmdzyh2kjsg0ihv7akj99f2q91g0jcj8-pub-package_info_plus-8.0.0/
[        ] [   +2 ms] Found plugin url_launcher at /nix/store/d8cg5dk9xwdwim8jwnvrw8nmijr3iv39-pub-url_launcher-6.3.0/
[        ] [   +2 ms] Found plugin url_launcher_macos at /nix/store/dwn48qwaid32b34iczyhm343fcyimd5d-pub-url_launcher_macos-3.2.0/
[        ] [   +1 ms] Found plugin url_launcher_ios at /nix/store/fcqj5jjjblla640qwp6fyss8q6zr5n7d-pub-url_launcher_ios-6.3.0/
[        ] [        ] Found plugin shared_preferences_web at /nix/store/fwhqzisbkvp08wgks38pda4yrha9viqb-pub-shared_preferences_web-2.3.0/
[        ] [  +21 ms] Found plugin shared_preferences at /nix/store/m33wwmrhdiwq9r6lym6nnrxk7xfcxd27-pub-shared_preferences-2.2.3/
[        ] [   +5 ms] Found plugin flutter_plugin_android_lifecycle at /nix/store/n3v1dangini95nmf8qlnlkvip2s5sd51-pub-flutter_plugin_android_lifecycle-2.0.20/
[        ] [   +4 ms] Found plugin url_launcher_android at /nix/store/nlqazhkh1xv1qda4s0mdzf6zb905lagd-pub-url_launcher_android-6.3.3/
[        ] [   +3 ms] Found plugin file_picker at /nix/store/pmsshjvm2a4mcyrbfcjfb3ynva9y3m8p-file_picker-8.0.6/
[        ] [   +1 ms] Found plugin shared_preferences_windows at /nix/store/qidlkg4vjwjmch6w595n8bziw4m9cml4-pub-shared_preferences_windows-2.3.2/
[        ] [   +4 ms] Found plugin path_provider_windows at /nix/store/wpx8wfbislwv8zzygrpnbj2yfhdpw0bc-pub-path_provider_windows-2.2.1/
[        ] [        ] Found plugin url_launcher_web at /nix/store/wqh6k1g8gqhi30ixlnn73cnymx2wimrj-pub-url_launcher_web-2.3.1/
[        ] [   +5 ms] Found plugin url_launcher_windows at /nix/store/zap879f3mvgncq3nnq9a0z57cq0n9gzr-pub-url_launcher_windows-3.1.1/
[        ] [   +5 ms] Package file_picker:linux references file_picker:linux as the default plugin, but it does not provide an inline implementation.
[        ]            Ask the maintainers of file_picker to either avoid referencing a default implementation via `platforms: linux: default_package: file_picker` or add an inline implementation to file_picker via `platforms: linux:` `pluginClass` or `dartPluginClass`.
[        ] [   +1 ms] Package file_picker:macos references file_picker:macos as the default plugin, but it does not provide an inline implementation.
[        ]            Ask the maintainers of file_picker to either avoid referencing a default implementation via `platforms: macos: default_package: file_picker` or add an inline implementation to file_picker via `platforms: macos:` `pluginClass` or `dartPluginClass`.
[        ] [        ] Package file_picker:windows references file_picker:windows as the default plugin, but it does not provide an inline implementation.
[        ]            Ask the maintainers of file_picker to either avoid referencing a default implementation via `platforms: windows: default_package: file_picker` or add an inline implementation to file_picker via `platforms: windows:` `pluginClass` or `dartPluginClass`.
[        ] [  +12 ms] gen_dart_plugin_registrant: Complete
[        ] [        ] kernel_snapshot_program: Starting due to {}
[   +2 ms] [   +5 ms] /nix/store/z1za1hwdayfcl4pgk6cxq2nmvy2bk5fi-flutter-wrapped-3.27.1-sdk-links/bin/cache/dart-sdk/bin/dartaotruntime /nix/store/z1za1hwdayfcl4pgk6cxq2nmvy2bk5fi-flutter-wrapped-3.27.1-sdk-links/bin/cache/dart-sdk/bin/snapshots/frontend_server_aot.dart.snapshot --sdk-root /nix/store/z1za1hwdayfcl4pgk6cxq2nmvy2bk5fi-flutter-wrapped-3.27.1-sdk-links/bin/cache/artifacts/engine/common/flutter_patched_sdk_product/ --target=flutter --no-print-incremental-dependencies -Ddart.vm.profile=false -Ddart.vm.product=true --delete-tostring-package-uri=dart:ui --delete-tostring-package-uri=package:flutter --aot --tfa --target-os linux --packages /build/source/.dart_tool/package_config.json --output-dill /build/source/.dart_tool/flutter_build/e50b9596a524fb6e714d2c428e5aea8d/program.dill --depfile /build/source/.dart_tool/flutter_build/e50b9596a524fb6e714d2c428e5aea8d/kernel_snapshot_program.d --source file:///build/source/.dart_tool/flutter_build/dart_plugin_registrant.dart --source package:flutter/src/dart_plugin_registrant.dart -Dflutter.dart_plugin_registrant=file:///build/source/.dart_tool/flutter_build/dart_plugin_registrant.dart --verbosity=error package:quickgui/main.dart
[        ] [+2760 ms] /nix/store/7iy1sgjphvl92c8xniyza9z0c346hj26-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1919:39: Error: Method not found: 'SchemeRainbow'.
[        ] [        ]       DynamicSchemeVariant.rainbow => SchemeRainbow(sourceColorHct: sourceColor, isDark: isDark, contrastLevel: contrastLevel),
[        ] [        ]                                       ^^^^^^^^^^^^^
[        ] [        ] /nix/store/7iy1sgjphvl92c8xniyza9z0c346hj26-flutter-sdk-flutter/lib/src/material/color_scheme.dart:1920:42: Error: Method not found: 'SchemeFruitSalad'.
[        ] [        ]       DynamicSchemeVariant.fruitSalad => SchemeFruitSalad(sourceColorHct: sourceColor, isDark: isDark, contrastLevel: contrastLevel),
[        ] [        ]                                          ^^^^^^^^^^^^^^^^
[        ] [+4433 ms] Persisting file store
[        ] [   +1 ms] Done persisting file store
[        ] [   +1 ms] Target kernel_snapshot_program failed: Exception
[        ]            #0      KernelSnapshotProgram.build (package:flutter_tools/src/build_system/targets/common.dart:274:7)
[        ]            <asynchronous suspension>
[        ]            #1      _BuildInstance._invokeInternal (package:flutter_tools/src/build_system/build_system.dart:891:9)
[        ]            <asynchronous suspension>
[        ]            #2      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
[        ]            <asynchronous suspension>
[        ]            #3      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:829:32)
[        ]            <asynchronous suspension>
[        ]            #4      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
[        ]            <asynchronous suspension>
[        ]            #5      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:829:32)
[        ]            <asynchronous suspension>
[        ]            #6      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
[   +2 ms]            <asynchronous suspension>
[        ]            #7      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:829:32)
[        ]            <asynchronous suspension>
[        ]            #8      Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
[        ]            <asynchronous suspension>
[        ]            #9      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:829:32)
[        ]            <asynchronous suspension>
[        ]            #10     Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
[        ]            <asynchronous suspension>
[        ]            #11     _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:829:32)
[        ]            <asynchronous suspension>
[        ]            #12     Future.wait.<anonymous closure> (dart:async/future.dart:520:21)
[        ]            <asynchronous suspension>
[        ]            #13     _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:829:32)
[        ]            <asynchronous suspension>
[        ]            #14     FlutterBuildSystem.build (package:flutter_tools/src/build_system/build_system.dart:651:16)
[        ]            <asynchronous suspension>
[        ]            #15     AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:329:32)
[        ]            <asynchronous suspension>
[        ]            #16     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1450:27)
[        ]            <asynchronous suspension>
[        ]            #17     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
[        ]            <asynchronous suspension>
[        ]            #18     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
[        ]            <asynchronous suspension>
[        ]            #19     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:417:9)
[        ]            <asynchronous suspension>
[        ]            #20     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
[        ]            <asynchronous suspension>
[        ]            #21     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
[        ]            <asynchronous suspension>
[        ]            #22     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:131:9)
[        ]            <asynchronous suspension>
[        ]            #23     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
[        ]            <asynchronous suspension>
[        ]            #24     main (package:flutter_tools/executable.dart:94:3)
[        ]            <asynchronous suspension>
[        ] [   +4 ms] "flutter assemble" took 7,563ms.
[        ] [   +1 ms] 
[        ]            #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
[        ]            #1      AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:346:7)
[        ]            <asynchronous suspension>
[        ]            #2      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1450:27)
[        ]            <asynchronous suspension>
[        ]            #3      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
[        ]            <asynchronous suspension>
[        ]            #4      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
[        ]            <asynchronous suspension>
[        ]            #5      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:417:9)
[        ]            <asynchronous suspension>
[        ]            #6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
[        ]            <asynchronous suspension>
[        ]            #7      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
[        ]            <asynchronous suspension>
[        ]            #8      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:131:9)
[        ]            <asynchronous suspension>
[        ]            #9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
[        ]            <asynchronous suspension>
[        ]            #10     main (package:flutter_tools/executable.dart:94:3)
[        ]            <asynchronous suspension>
[        ] [        ] Running 1 shutdown hook
[        ] [        ] Shutdown hooks complete
[        ] [   +1 ms] exiting with code 1
[        ] ninja: build stopped: subcommand failed.
[   +7 ms] Building Linux application... (completed in 9.2s)
[        ] "flutter linux" took 9,494ms.
[   +1 ms] Build process failed
[        ] 
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      _runBuild (package:flutter_tools/src/linux/build_linux.dart:210:5)
           <asynchronous suspension>
           #2      buildLinux (package:flutter_tools/src/linux/build_linux.dart:83:5)
           <asynchronous suspension>
           #3      BuildLinuxCommand.runCommand (package:flutter_tools/src/commands/build_linux.dart:87:5)
           <asynchronous suspension>
           #4      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1450:27)
           <asynchronous suspension>
           #5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
           <asynchronous suspension>
           #6      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
           <asynchronous suspension>
           #7      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:417:9)
           <asynchronous suspension>
           #8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
           <asynchronous suspension>
           #9      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
           <asynchronous suspension>
           #10     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:131:9)
           <asynchronous suspension>
           #11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
           <asynchronous suspension>
           #12     main (package:flutter_tools/executable.dart:94:3)
           <asynchronous suspension>
           
           
[        ] Running 1 shutdown hook
[        ] Shutdown hooks complete
[   +2 ms] exiting with code 1
error: builder for '/nix/store/q7c6p0y139f4srhgz2gsjv22ifdh5mrg-quickgui-1.2.10-unstable-2024-12-30.drv' failed with exit code 1;
       last 25 log lines:
       >            #3      BuildLinuxCommand.runCommand (package:flutter_tools/src/commands/build_linux.dart:87:5)
       >            <asynchronous suspension>
       >            #4      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1450:27)
       >            <asynchronous suspension>
       >            #5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
       >            <asynchronous suspension>
       >            #6      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
       >            <asynchronous suspension>
       >            #7      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:417:9)
       >            <asynchronous suspension>
       >            #8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
       >            <asynchronous suspension>
       >            #9      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
       >            <asynchronous suspension>
       >            #10     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:131:9)
       >            <asynchronous suspension>
       >            #11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
       >            <asynchronous suspension>
       >            #12     main (package:flutter_tools/executable.dart:94:3)
       >            <asynchronous suspension>
       >            
       >            
       > [        ] Running 1 shutdown hook
       > [        ] Shutdown hooks complete
       > [   +2 ms] exiting with code 1
```

--- a/pkgs/by-name/qu/quickgui/pubspec.lock.json
+++ b/pkgs/by-name/qu/quickgui/pubspec.lock.json
@@ -4,21 +4,21 @@
       "dependency": "transitive",
       "description": {
         "name": "_discoveryapis_commons",
-        "sha256": "f8bb1fdbd77f3d5c1d62b5b0eca75fbf1e41bf4f6c62628f880582e2182ae45d",
+        "sha256": "113c4100b90a5b70a983541782431b82168b3cae166ab130649c36eb3559d498",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.6"
+      "version": "1.0.7"
     },
     "ansicolor": {
       "dependency": "transitive",
       "description": {
         "name": "ansicolor",
-        "sha256": "8bf17a8ff6ea17499e40a2d2542c2f481cd7615760c6d34065cb22bfd22e6880",
+        "sha256": "50e982d500bc863e1d703448afdbf9e5a72eb48840a4f766fa361ffd6877055f",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.0.2"
+      "version": "2.0.3"
     },
     "archive": {
       "dependency": "transitive",
@@ -34,11 +34,11 @@
       "dependency": "transitive",
       "description": {
         "name": "args",
-        "sha256": "7cf60b9f0cc88203c5a190b4cd62a99feea42759a7fa695010eb5de1c0b2252a",
+        "sha256": "bf9f5caeea8d8fe6721a9c358dd8a5c1947b27f1cfaa18b39c301273594919e6",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.5.0"
+      "version": "2.6.0"
     },
     "async": {
       "dependency": "transitive",
@@ -94,11 +94,11 @@
       "dependency": "transitive",
       "description": {
         "name": "cli_util",
-        "sha256": "c05b7406fdabc7a49a3929d4af76bcaccbbffcbcdcf185b082e1ae07da323d19",
+        "sha256": "ff6785f7e9e3c38ac98b2fb035701789de90154024a75b6cb926445e83197d1c",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.4.1"
+      "version": "0.4.2"
     },
     "clock": {
       "dependency": "transitive",
@@ -134,21 +134,21 @@
       "dependency": "transitive",
       "description": {
         "name": "cross_file",
-        "sha256": "55d7b444feb71301ef6b8838dbc1ae02e63dd48c8773f3810ff53bb1e2945b32",
+        "sha256": "7caf6a750a0c04effbb52a676dce9a4a592e10ad35c34d6d2d0e4811160d5670",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.3.4+1"
+      "version": "0.3.4+2"
     },
     "crypto": {
       "dependency": "transitive",
       "description": {
         "name": "crypto",
-        "sha256": "ff625774173754681d66daaf4a448684fb04b78f902da9cb3d308c19cc5e8bab",
+        "sha256": "1e445881f28f22d6140f181e07737b22f1e099a5e1ff94b0af2f9e4a463f4855",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.0.3"
+      "version": "3.0.6"
     },
     "cupertino_icons": {
       "dependency": "direct main",
@@ -184,11 +184,21 @@
       "dependency": "transitive",
       "description": {
         "name": "dio",
-        "sha256": "11e40df547d418cc0c4900a9318b26304e665da6fa4755399a9ff9efd09034b5",
+        "sha256": "5598aa796bbf4699afd5c67c0f5f6e2ed542afc956884b9cd58c306966efc260",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "5.4.3+1"
+      "version": "5.7.0"
+    },
+    "dio_web_adapter": {
+      "dependency": "transitive",
+      "description": {
+        "name": "dio_web_adapter",
+        "sha256": "33259a9276d6cea88774a0000cfae0d861003497755969c92faa223108620dc8",
+        "url": "https://pub.dev"
+      },
+      "source": "hosted",
+      "version": "2.0.0"
     },
     "fake_async": {
       "dependency": "transitive",
@@ -204,31 +214,31 @@
       "dependency": "transitive",
       "description": {
         "name": "ffi",
-        "sha256": "493f37e7df1804778ff3a53bd691d8692ddf69702cf4c1c1096a2e41b4779e21",
+        "sha256": "16ed7b077ef01ad6170a3d0c57caa4a112a38d7a2ed5602e0aca9ca6f3d98da6",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.1.2"
+      "version": "2.1.3"
     },
     "file": {
       "dependency": "transitive",
       "description": {
         "name": "file",
-        "sha256": "5fc22d7c25582e38ad9a8515372cd9a93834027aacf1801cf01164dac0ffa08c",
+        "sha256": "a3b4f84adafef897088c160faf7dfffb7696046cb13ae90b508c2cbc95d3b8d4",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "7.0.0"
+      "version": "7.0.1"
     },
     "file_picker": {
       "dependency": "direct main",
       "description": {
         "name": "file_picker",
-        "sha256": "824f5b9f389bfc4dddac3dea76cd70c51092d9dff0b2ece7ef4f53db8547d258",
+        "sha256": "16dc141db5a2ccc6520ebb6a2eb5945b1b09e95085c021d9f914f8ded7f1465c",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "8.0.6"
+      "version": "8.1.4"
     },
     "flutter": {
       "dependency": "direct main",
@@ -296,21 +306,21 @@
       "dependency": "transitive",
       "description": {
         "name": "flutter_plugin_android_lifecycle",
-        "sha256": "c6b0b4c05c458e1c01ad9bcc14041dd7b1f6783d487be4386f793f47a8a4d03e",
+        "sha256": "9b78450b89f059e96c9ebb355fa6b3df1d6b330436e0b885fb49594c41721398",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.0.20"
+      "version": "2.0.23"
     },
     "flutter_svg": {
       "dependency": "direct main",
       "description": {
         "name": "flutter_svg",
-        "sha256": "7b4ca6cf3304575fe9c8ec64813c8d02ee41d2afe60bcfe0678bcb5375d596a2",
+        "sha256": "54900a1a1243f3c4a5506d853a2b5c2dbc38d5f27e52a52618a8054401431123",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.0.10+1"
+      "version": "2.0.16"
     },
     "flutter_test": {
       "dependency": "direct dev",
@@ -378,11 +388,11 @@
       "dependency": "transitive",
       "description": {
         "name": "google_identity_services_web",
-        "sha256": "9482364c9f8b7bd36902572ebc3a7c2b5c8ee57a9c93e6eb5099c1a9ec5265d8",
+        "sha256": "55580f436822d64c8ff9a77e37d61f5fb1e6c7ec9d632a43ee324e2a05c3c6c9",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.3.1+1"
+      "version": "0.3.3"
     },
     "googleapis": {
       "dependency": "transitive",
@@ -408,11 +418,11 @@
       "dependency": "transitive",
       "description": {
         "name": "http",
-        "sha256": "761a297c042deedc1ffbb156d6e2af13886bb305c2a343a4d972504cd67dd938",
+        "sha256": "b9c29a161230ee03d3ccf545097fccd9b87a5264228c5d348202e0f0c28f9010",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.2.1"
+      "version": "1.2.2"
     },
     "http_parser": {
       "dependency": "transitive",
@@ -428,11 +438,11 @@
       "dependency": "transitive",
       "description": {
         "name": "image",
-        "sha256": "2237616a36c0d69aef7549ab439b833fb7f9fb9fc861af2cc9ac3eedddd69ca8",
+        "sha256": "f31d52537dc417fdcde36088fdf11d191026fd5e4fae742491ebd40e5a8bea7d",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "4.2.0"
+      "version": "4.3.0"
     },
     "intl": {
       "dependency": "transitive",
@@ -468,21 +478,21 @@
       "dependency": "transitive",
       "description": {
         "name": "leak_tracker",
-        "sha256": "7f0df31977cb2c0b88585095d168e689669a2cc9b97c309665e3386f3e9d341a",
+        "sha256": "3f87a60e8c63aecc975dda1ceedbc8f24de75f09e4856ea27daf8958f2f0ce05",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "10.0.4"
+      "version": "10.0.5"
     },
     "leak_tracker_flutter_testing": {
       "dependency": "transitive",
       "description": {
         "name": "leak_tracker_flutter_testing",
-        "sha256": "06e98f569d004c1315b991ded39924b21af84cf14cc94791b8aea337d25b57f8",
+        "sha256": "932549fb305594d82d7183ecd9fa93463e9914e1b67cacc34bc40906594a1806",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.0.3"
+      "version": "3.0.5"
     },
     "leak_tracker_testing": {
       "dependency": "transitive",
@@ -518,11 +528,11 @@
       "dependency": "transitive",
       "description": {
         "name": "logging",
-        "sha256": "623a88c9594aa774443aa3eb2d41807a48486b5613e67599fb4c41c0ad47c340",
+        "sha256": "c8245ada5f1717ed44271ed1c26b8ce85ca3228fd2ffdb75468ab01979309d61",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.2.0"
+      "version": "1.3.0"
     },
     "matcher": {
       "dependency": "transitive",
@@ -538,31 +548,31 @@
       "dependency": "transitive",
       "description": {
         "name": "material_color_utilities",
-        "sha256": "0e0a020085b65b6083975e499759762399b4475f766c21668c4ecca34ea74e5a",
+        "sha256": "f7142bb1154231d7ea5f96bc7bde4bda2a0945d2806bb11670e30b850d56bdec",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.8.0"
+      "version": "0.11.1"
     },
     "meta": {
       "dependency": "transitive",
       "description": {
         "name": "meta",
-        "sha256": "7687075e408b093f36e6bbf6c91878cc0d4cd10f409506f7bc996f68220b9136",
+        "sha256": "bdb68674043280c3428e9ec998512fb681678676b3c54e773629ffe74419f8c7",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.12.0"
+      "version": "1.15.0"
     },
     "msix": {
       "dependency": "transitive",
       "description": {
         "name": "msix",
-        "sha256": "519b183d15dc9f9c594f247e2d2339d855cf0eaacc30e19b128e14f3ecc62047",
+        "sha256": "c50d6bd1aafe0d071a3c1e5a5ccb056404502935cb0a549e3178c4aae16caf33",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.16.7"
+      "version": "3.16.8"
     },
     "mustache_template": {
       "dependency": "transitive",
@@ -598,21 +608,21 @@
       "dependency": "direct main",
       "description": {
         "name": "package_info_plus",
-        "sha256": "b93d8b4d624b4ea19b0a5a208b2d6eff06004bc3ce74c06040b120eeadd00ce0",
+        "sha256": "da8d9ac8c4b1df253d1a328b7bf01ae77ef132833479ab40763334db13b91cce",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "8.0.0"
+      "version": "8.1.1"
     },
     "package_info_plus_platform_interface": {
       "dependency": "transitive",
       "description": {
         "name": "package_info_plus_platform_interface",
-        "sha256": "f49918f3433a3146047372f9d4f1f847511f2acd5cd030e1f44fe5a50036b70e",
+        "sha256": "ac1f4a4847f1ade8e6a87d1f39f5d7c67490738642e2542f559ec38c37489a66",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.0.0"
+      "version": "3.0.1"
     },
     "parse_app_package": {
       "dependency": "transitive",
@@ -638,11 +648,11 @@
       "dependency": "transitive",
       "description": {
         "name": "path_parsing",
-        "sha256": "e3e67b1629e6f7e8100b367d3db6ba6af4b1f0bb80f64db18ef1fbabd2fa9ccf",
+        "sha256": "883402936929eac138ee0a45da5b0f2c80f89913e6dc3bf77eb65b84b409c6ca",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.1"
+      "version": "1.1.0"
     },
     "path_provider_linux": {
       "dependency": "transitive",
@@ -668,11 +678,11 @@
       "dependency": "transitive",
       "description": {
         "name": "path_provider_windows",
-        "sha256": "8bc9f22eee8690981c22aa7fc602f5c85b497a6fb2ceb35ee5a5e5ed85ad8170",
+        "sha256": "bd6f00dbd873bfb70d0761682da2b3a2c2fccc2b9e84c495821639601d81afe7",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.2.1"
+      "version": "2.3.0"
     },
     "petitparser": {
       "dependency": "transitive",
@@ -688,11 +698,11 @@
       "dependency": "transitive",
       "description": {
         "name": "platform",
-        "sha256": "9b71283fc13df574056616011fb138fd3b793ea47cc509c189a6c3fa5f8a1a65",
+        "sha256": "5d6b1b0036a5f331ebc77c850ebc8506cbc1e9416c27e59b439f917a902a4984",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.5"
+      "version": "3.1.6"
     },
     "plist_parser": {
       "dependency": "transitive",
@@ -718,11 +728,11 @@
       "dependency": "direct main",
       "description": {
         "name": "process_run",
-        "sha256": "6052115540ad88715d6bcee60656970f70c68c85846d1948b92e435f0382899e",
+        "sha256": "a68fa9727392edad97a2a96a77ce8b0c17d28336ba1b284b1dfac9595a4299ea",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.0+1"
+      "version": "1.2.2+1"
     },
     "provider": {
       "dependency": "direct main",
@@ -768,11 +778,11 @@
       "dependency": "direct main",
       "description": {
         "name": "quiver",
-        "sha256": "b1c1ac5ce6688d77f65f3375a9abb9319b3cb32486bdc7a1e0fdf004d7ba4e47",
+        "sha256": "ea0b925899e64ecdfbf9c7becb60d5b50e706ade44a85b2363be2a22d88117d2",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.2.1"
+      "version": "3.2.2"
     },
     "recase": {
       "dependency": "transitive",
@@ -788,71 +798,71 @@
       "dependency": "direct main",
       "description": {
         "name": "shared_preferences",
-        "sha256": "d3bbe5553a986e83980916ded2f0b435ef2e1893dfaa29d5a7a790d0eca12180",
+        "sha256": "95f9997ca1fb9799d494d0cb2a780fd7be075818d59f00c43832ed112b158a82",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.2.3"
+      "version": "2.3.3"
     },
     "shared_preferences_android": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_android",
-        "sha256": "93d0ec9dd902d85f326068e6a899487d1f65ffcd5798721a95330b26c8131577",
+        "sha256": "7f172d1b06de5da47b6264c2692ee2ead20bbbc246690427cdb4fc301cd0c549",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.2.3"
+      "version": "2.3.4"
     },
     "shared_preferences_foundation": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_foundation",
-        "sha256": "0a8a893bf4fd1152f93fec03a415d11c27c74454d96e2318a7ac38dd18683ab7",
+        "sha256": "07e050c7cd39bad516f8d64c455f04508d09df104be326d8c02551590a0d513d",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.4.0"
+      "version": "2.5.3"
     },
     "shared_preferences_linux": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_linux",
-        "sha256": "9f2cbcf46d4270ea8be39fa156d86379077c8a5228d9dfdb1164ae0bb93f1faa",
+        "sha256": "580abfd40f415611503cae30adf626e6656dfb2f0cee8f465ece7b6defb40f2f",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.2"
+      "version": "2.4.1"
     },
     "shared_preferences_platform_interface": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_platform_interface",
-        "sha256": "22e2ecac9419b4246d7c22bfbbda589e3acf5c0351137d87dd2939d984d37c3b",
+        "sha256": "57cbf196c486bc2cf1f02b85784932c6094376284b3ad5779d1b1c6c6a816b80",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.2"
+      "version": "2.4.1"
     },
     "shared_preferences_web": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_web",
-        "sha256": "9aee1089b36bd2aafe06582b7d7817fd317ef05fc30e6ba14bff247d0933042a",
+        "sha256": "d2ca4132d3946fec2184261726b355836a82c33d7d5b67af32692aff18a4684e",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.0"
+      "version": "2.4.2"
     },
     "shared_preferences_windows": {
       "dependency": "transitive",
       "description": {
         "name": "shared_preferences_windows",
-        "sha256": "841ad54f3c8381c480d0c9b508b89a34036f512482c407e6df7a9c4aa2ef8f59",
+        "sha256": "94ef0f72b2d71bc3e700e025db3710911bd51a71cefb65cc609dd0d9a982e3c1",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.2"
+      "version": "2.4.1"
     },
     "shell_executor": {
       "dependency": "transitive",
@@ -924,11 +934,11 @@
       "dependency": "transitive",
       "description": {
         "name": "synchronized",
-        "sha256": "539ef412b170d65ecdafd780f924e5be3f60032a1128df156adad6c5b373d558",
+        "sha256": "69fe30f3a8b04a0be0c15ae6490fc859a78ef4c43ae2dd5e8a623d45bfcf9225",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.0+1"
+      "version": "3.3.0+3"
     },
     "term_glyph": {
       "dependency": "transitive",
@@ -944,11 +954,11 @@
       "dependency": "transitive",
       "description": {
         "name": "test_api",
-        "sha256": "9955ae474176f7ac8ee4e989dadfb411a58c30415bcfb648fa04b2b8a03afa7f",
+        "sha256": "5b8a98dafc4d5c4c9c72d8b31ab2b23fc13422348d2997120294d3bac86b4ddb",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.7.0"
+      "version": "0.7.2"
     },
     "tuple": {
       "dependency": "direct main",
@@ -964,61 +974,61 @@
       "dependency": "transitive",
       "description": {
         "name": "typed_data",
-        "sha256": "facc8d6582f16042dd49f2463ff1bd6e2c9ef9f3d5da3d9b087e244a7b564b3c",
+        "sha256": "f9049c039ebfeb4cf7a7104a675823cd72dba8297f264b6637062516699fa006",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.3.2"
+      "version": "1.4.0"
     },
     "url_launcher": {
       "dependency": "direct main",
       "description": {
         "name": "url_launcher",
-        "sha256": "21b704ce5fa560ea9f3b525b43601c678728ba46725bab9b01187b4831377ed3",
+        "sha256": "9d06212b1362abc2f0f0d78e6f09f726608c74e3b9462e8368bb03314aa8d603",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "6.3.0"
+      "version": "6.3.1"
     },
     "url_launcher_android": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_android",
-        "sha256": "ceb2625f0c24ade6ef6778d1de0b2e44f2db71fded235eb52295247feba8c5cf",
+        "sha256": "6fc2f56536ee873eeb867ad176ae15f304ccccc357848b351f6f0d8d4a40d193",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "6.3.3"
+      "version": "6.3.14"
     },
     "url_launcher_ios": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_ios",
-        "sha256": "7068716403343f6ba4969b4173cbf3b84fc768042124bc2c011e5d782b24fe89",
+        "sha256": "e43b677296fadce447e987a2f519dcf5f6d1e527dc35d01ffab4fff5b8a7063e",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "6.3.0"
+      "version": "6.3.1"
     },
     "url_launcher_linux": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_linux",
-        "sha256": "ab360eb661f8879369acac07b6bb3ff09d9471155357da8443fd5d3cf7363811",
+        "sha256": "4e9ba368772369e3e08f231d2301b4ef72b9ff87c31192ef471b380ef29a4935",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.1"
+      "version": "3.2.1"
     },
     "url_launcher_macos": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_macos",
-        "sha256": "9a1a42d5d2d95400c795b2914c36fdcb525870c752569438e4ebb09a2b5d90de",
+        "sha256": "769549c999acdb42b8bcfa7c43d72bf79a382ca7441ab18a808e101149daf672",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.2.0"
+      "version": "3.2.1"
     },
     "url_launcher_platform_interface": {
       "dependency": "transitive",
@@ -1034,21 +1044,21 @@
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_web",
-        "sha256": "8d9e750d8c9338601e709cd0885f95825086bd8b642547f26bda435aade95d8a",
+        "sha256": "772638d3b34c779ede05ba3d38af34657a05ac55b06279ea6edd409e323dca8e",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "2.3.1"
+      "version": "2.3.3"
     },
     "url_launcher_windows": {
       "dependency": "transitive",
       "description": {
         "name": "url_launcher_windows",
-        "sha256": "ecf9725510600aa2bb6d7ddabe16357691b6d2805f66216a97d1b881e21beff7",
+        "sha256": "44cf3aabcedde30f2dba119a9dea3b0f2672fbe6fa96e85536251d678216b3c4",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "3.1.1"
+      "version": "3.1.3"
     },
     "uuid": {
       "dependency": "transitive",
@@ -1064,31 +1074,31 @@
       "dependency": "transitive",
       "description": {
         "name": "vector_graphics",
-        "sha256": "32c3c684e02f9bc0afb0ae0aa653337a2fe022e8ab064bcd7ffda27a74e288e3",
+        "sha256": "27d5fefe86fb9aace4a9f8375b56b3c292b64d8c04510df230f849850d912cb7",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.1.11+1"
+      "version": "1.1.15"
     },
     "vector_graphics_codec": {
       "dependency": "transitive",
       "description": {
         "name": "vector_graphics_codec",
-        "sha256": "c86987475f162fadff579e7320c7ddda04cd2fdeffbe1129227a85d9ac9e03da",
+        "sha256": "2430b973a4ca3c4dbc9999b62b8c719a160100dcbae5c819bae0cacce32c9cdb",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.1.11+1"
+      "version": "1.1.12"
     },
     "vector_graphics_compiler": {
       "dependency": "transitive",
       "description": {
         "name": "vector_graphics_compiler",
-        "sha256": "12faff3f73b1741a36ca7e31b292ddeb629af819ca9efe9953b70bd63fc8cd81",
+        "sha256": "1b4b9e706a10294258727674a340ae0d6e64a7231980f9f9a3d12e4b42407aad",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.1.11+1"
+      "version": "1.1.16"
     },
     "vector_math": {
       "dependency": "transitive",
@@ -1114,31 +1124,31 @@
       "dependency": "transitive",
       "description": {
         "name": "vm_service",
-        "sha256": "3923c89304b715fb1eb6423f017651664a03bf5f4b29983627c4da791f74a4ec",
+        "sha256": "5c5f338a667b4c644744b661f309fb8080bb94b18a7e91ef1dbd343bed00ed6d",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "14.2.1"
+      "version": "14.2.5"
     },
     "web": {
       "dependency": "transitive",
       "description": {
         "name": "web",
-        "sha256": "97da13628db363c635202ad97068d47c5b8aa555808e7a9411963c533b449b27",
+        "sha256": "cd3543bd5798f6ad290ea73d210f423502e71900302dde696f8bff84bf89a1cb",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "0.5.1"
+      "version": "1.1.0"
     },
     "win32": {
       "dependency": "transitive",
       "description": {
         "name": "win32",
-        "sha256": "a79dbe579cb51ecd6d30b17e0cae4e0ea15e2c0e66f69ad4198f22a6789e94f4",
+        "sha256": "8b338d4486ab3fbc0ba0db9f9b4f5239b6697fcee427939a40e720cbb9ee0a69",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "5.5.1"
+      "version": "5.9.0"
     },
     "window_size": {
       "dependency": "direct main",
@@ -1155,11 +1165,11 @@
       "dependency": "transitive",
       "description": {
         "name": "xdg_directories",
-        "sha256": "faea9dee56b520b55a566385b84f2e8de55e7496104adada9962e0bd11bcff1d",
+        "sha256": "7a3f37b05d989967cdddcbb571f1ea834867ae2faa29725fd085180e0883aa15",
         "url": "https://pub.dev"
       },
       "source": "hosted",
-      "version": "1.0.4"
+      "version": "1.1.0"
     },
     "xml": {
       "dependency": "transitive",
@@ -1183,7 +1193,7 @@
     }
   },
   "sdks": {
-    "dart": ">=3.4.0 <4.0.0",
-    "flutter": ">=3.22.0"
+    "dart": ">=3.5.0 <4.0.0",
+    "flutter": ">=3.24.0"
   }
 }

